src/java.base/share/classes/java/util/concurrent/Flow.java

```diff
@@ -60,7 +60,8 @@
  * TRUE} item to a single subscriber.  Because the subscriber receives
  * only a single item, this class does not use buffering and ordering
  * control required in most implementations (for example {@link
- * SubmissionPublisher}).
+ * SubmissionPublisher}), and omits some error processing needed to
+ * fully conform to ReactiveStream rules.
```

Comment:
I had to re-read the comments on the relevant [issue] from 2018; thanks for fixing it.

Nit: ReactiveStream. Earlier in that doc comment, it is referred to as "reactive-streams". We should choose one of those.

We can say reactive-streams or Reactive Streams. The latter seems to be the term of choice at https://github.com/reactive-streams/reactive-streams-jvm/.

@viktorklang-ora, thoughts?

[issue]: https://bugs.openjdk.org/browse/JDK-8210149

Reply:
@pavelrappo Indeed, "Reactive Streams" would be better. Perhaps something like this:

```suggestion
 * fully conform to the Reactive Streams specification.
```